packages/appeals-service-api/src/lib/questionnaire-due-date.js

```diff
@@ -0,0 +1,36 @@
+const { addDays, endOfDay, format } = require('date-fns');
+
+module.exports = {
+  /**
+   * Given an appeal object, return the due date for the Questionnaire.
+   *
+   * At present, there is only one appeal type - Householder. In future there will be more and this
+   * logic will need to be updated. At the time of creation, `appeal.type` is not a 'thing', and so
+   * we have no easy (i.e. non-brittle) way of determining what type of appeal we are dealing with.
+   *
+   * <b style="color: red;">Therefore all appeals are currently considered as Householder</b>
+   *
+   * @param appeal the appeal object
+   * @returns {Date}
+   */
+  getQuestionnaireDueDate: (appeal) => {
+    const householderAppealQuestionnaireAdditionalStartDays = 5;
+
+    const dueDate = addDays(
+      appeal.submissionDate,
+      householderAppealQuestionnaireAdditionalStartDays
+    );
+
+    return endOfDay(dueDate);
+  },
+
+  /**
+   * Given an appeal object, return the formatted due date for the Questionnaire.
+   *
+   * @param appeal the appeal object
+   * @param dateFormat an optional date format string - https://date-fns.org/v2.22.1/docs/format
+   * @returns {string}
+   */
+  getFormattedQuestionnaireDueDate: (appeal, dateFormat = 'dd MMMM yyyy') =>
```

Comment:
As above.

Reply:
it's called here:

https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/1301/files#diff-1da782b939b62689d49ff8fbc93d83f5ca9562300b16c6b5dba31507fa3cb225R206